### ISSUE TITLE
Add Terraform link to any specific service

### ISSUE THREAD

**Issue by Roboe77** [opentelekomcloud/otcdocstheme#272]
Hi,

we have a general overview page about Terraform in our docs pages: https://docs.otc.t-systems.com/developer/iac.htmlsind
Nevertheless, we got a customer feedback to link each service directly to terraform. Example of how it can look like in the attached screenshot:

![fcf2debf-72b2-4249-a008-71d8dd10f605](https://github.com/user-attachments/assets/2835863d-47b7-441b-b35a-2302505a6216)

Would it be possible to add this?

Appreciated.

**Comment by tischrei** [comment]
No, we won't do that. There is not a good way to combine Terraform resources to one services which containes multiple ones. There is no single link from Terraform side to combine that in a usable way.
